system-tests/tests/load/cre/workflow_don_load_test.go

```diff
@@ -412,34 +412,65 @@ func TestLoad_Workflow_Streams_MockCapabilities(t *testing.T) {
 	// run the load
 	generator.Run(true)
 
-	tag := "local-test"
+	tag := "local-test-" + time.Now().Format("20060102150405")
 	if os.Getenv("CI") == "true" {
 		// When running in CI, use the GitHub commit SHA
 		commitSHA := os.Getenv("GITHUB_SHA")
 		if commitSHA != "" {
-			tag = commitSHA
+			tag = commitSHA + time.Now().Format("20060102150405")
 		}
 	} else if gitSHA := os.Getenv("GITHUB_SHA"); gitSHA != "" {
 		// For local runs with manually set GITHUB_SHA
 		tag = gitSHA
 	}
 
+	promConfig := benchspy.NewPrometheusConfig()
+
+	prometheusExecutor, err := benchspy.NewPrometheusQueryExecutor(
+		map[string]string{
+			"cpu_percent":          `avg (rate(container_cpu_usage_seconds_total{name=~"workflow-node[1-9]"}[5m]) * 100)`,
```

Comment:
why not make this `workflow-node[1-9]` dynamic and dependent on the number of nodes in that DON?

Reply:
Updated the regex to match from 1 to 99